### PR TITLE
Recaptcha verification on contact and apply for jobs page

### PR DESCRIPTION
**Pull request by cp-shruti-s**
- Add Google score-based ReCaptcha verification in contact and apply for the jobs page
- Fix response of SendMail APIs
- Fix unit tests
- Update readme

### CODE REVIEW

File: vue-frontend/src/components/contact/ContactForm.vue [hunk 1]
```diff
@@ -258,13 +260,39 @@ export default {
...
   },
   methods: {
+    onloadCallback() {
+      grecaptcha.enterprise.ready(function () {
+        grecaptcha.enterprise
+          .execute("SITE_KEY", {
+            action: "login",
+          })
+          .then(function (token) {
+            console.log(token);
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/contact/ContactForm.vue:278]: Are you getting token?

Comment (cp-shruti-s) [reply vue-frontend/src/components/contact/ContactForm.vue:278]: yes

File: contact/contact.go [hunk 2]
```diff
@@ -43,6 +48,13 @@ func New(templateFs embed.FS, emailRepo utils.EmailRepository) *Template {
 func (repository *Template) SendContactMail(c *gin.Context) {
 	var input ContactDetails
 
+	projectID := "PROJECT_ID"
+	recaptchaSiteKey := "SITE_KEY"
+	token := "TOKEN"
+	recaptchaAction := "RECAPTCHA_ACTION"
+
+	createAssessment(projectID, recaptchaSiteKey, token, recaptchaAction)
```
Comment (cp-sumi-k) [inline contact/contact.go:56]: No need of this assessment for now. Just get the token from contact form to this api

File: vue-frontend/src/components/contact/ContactForm.vue (hunk 1 above)
Comment (cp-sumi-k) [reply vue-frontend/src/components/contact/ContactForm.vue:278]: Just pass it to API and verify it using enterprise API

Comment (cp-shruti-s) [reply vue-frontend/src/components/contact/ContactForm.vue:278]: Achha okay i try

**Review by cp-priyanka-g — APPROVED** [review]

**Review by cp-sumi-k — APPROVED** [review]

**Pull request merged**